### PR TITLE
Books feature

### PR DESCRIPTION
**Pull request by AleksandarIvezic**
### In this project we:
- Created a collection that keeps a list of books
- Created a function to add a new book to the collection, with title and author
- Created a function to remove a book from the collection
- Displayed all books saved in the collection in the top part of the page
- Made sure that when a user clicks the "Add" button:
  -  A new book is added to the collection
  - The new book is displayed on the page
- Made sure that when a user clicks the "Remove" button:
   - The correct book is removed from the collection
   - The correct book disappears from the page
- Made sure that data is preserved in the browser's memory by using localStorage

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+}
+function addBook() {
+  const title = document.getElementById('title');
+  const author = document.getElementById('author');
+  const book = new FormBook(title.value, author.value);
+  books.push(book);
+  saveBooks();
+  display();
+}
+addButton.onclick = addBook;
```
Comment (Lameck1) [inline script.js:43]: - [ ] [OPTIONAL] For good user experience, I suggest we clear the input fields after submitting a book to localStorage. I would also like to suggest, that we test input values first before submission so that we don't submit an empty book object to local storage. You can also go extra and make a submission of the same book title and invalid submission. Consider this in the next iteration of the app.

File: script.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    remove.textContent = 'Remove';
+    remove.onclick = (e) => {
+      const bookTitle = e.target.parentElement.childNodes[0].textContent;
+      const bookAuthor = e.target.parentElement.childNodes[1].textContent;
+      books = books.filter((book) => book.title !== bookTitle || book.author !== bookAuthor);
+      saveBooks();
+      display();
+    };
+    bookTitle.innerText = book.title;
+    bookAuthor.innerText = book.author;
```
Comment (Lameck1) [inline script.js:28]: - [ ] [OPTIONAL] It is highly recommended that we use object Destructuring with ES6. Instead of using the dot notation, we can do something like this:
```JavaScript
const { title, author } = book;
```

And then use the variables in our code.

**Review by Lameck1 — APPROVED** [review]
Hi, @ShinobiWarior & @emmanuellekamwa

You both have done well in this task. Consider your project complete!  It's time to merge it :shipit: 


Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
